Sources/Sentry/SentryClient.m

```diff
@@ -320,6 +321,12 @@ - (SentryEvent *_Nullable)prepareEvent:(SentryEvent *)event
 
     event = [scope applyToEvent:event maxBreadcrumb:self.options.maxBreadcrumbs];
 
+    // With scope applied, before running callbacks run:
```

Comment:
`l`: I think we don't really need this comment here.

Reply:
You might notice that we assign environment above in this method too. So I wanted to be clear that this is "not up to be refactored". The intent is to only get this value after scopeApply is done.